security/pkg/server/monitoring/monitoring.go

```diff
@@ -1,120 +0,0 @@
-// Copyright 2019 Istio Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package monitoring
```

Comment:
Why removing this one? The monitoring can be used for istiod side-signing API as well.

Reply:
We don't need two metrics servers. I didn't delete any metrics, just the server. Any metrics citadel exposes will be served on istiod monitoring server - you can see this already today. Although most of the metrics are not useful, I don't think we get proper CSR metrics on the API we use in Istiod